Makefile

```diff
@@ -18,9 +18,15 @@ coverage:
 	coverage combine || true
 	coverage report
 
+.PHONY: coverage-lcov
+coverage-lcov:
+	coverage combine || true
+	coverage lcov -o coverage.info
+
 .PHONY: lint
 lint:
 	flake8 djedi
+	isort djedi
```

Comment:
We should call `pre-commit` here instead. Maybe there are other commands as well?

Reply:
Maybe, doesn't really matter to me.

I forgot that we should of course run pre-commit in CI as well

Reply:
Done!